pkgs/development/python-modules/mdutils/default.nix

```
@@ -7,14 +7,14 @@
 
 buildPythonPackage rec {
   pname = "mdutils";
-  version = "1.6.0";
+  version = "1.7.0";
   format = "setuptools";
```

```suggestion
  pyproject = true;
```
and set `build-system = [ setuptools ]`